bins/recipe-runner/src/runner.c

```diff
@@ -605,7 +607,12 @@ GglError runner(const RecipeRunnerArgs *args) {
         &resp
     );
     switch (ret) {
+    case GGL_ERR_NOMEM:
+        GGL_LOGW("Failed to get network proxy url from config - lack of memory."
+        );
+        break;
```

Comment:
This should probably exit with error. User set a proxy, intending for components to run with it, but we failed to apply it. Log should also be error.

Reply:
Agreed here! I completely missed this on my first review - I was too focused on the other error logs.

Reply:
Done.